python/README.md

```diff
@@ -0,0 +1,64 @@
+# Installation
+
+To use the `aida_interchange` package within your project you must first install it using the instructions described in the [Install setup.py](install-setup.py) section. It is recommended that you install `aida_interchange` into a virtual environment for development purposes. To install `aida_interchange`  within your virtual environment first activate your virtual environment and run the install command below. See [Python Development Setup](python-development-setup) for more details on creating a virtual environment.
+
+# Install setup.py
+
+To install `aida_interchange`, run the following command within the `python` directory:
+
+```bash
+python3 setup.py install
+```
+The `aida_interchange` modules can now be imported into your project.
+
+# Python Development Setup
+
+Setting up a virtual environment for Python development is the recommended best way to mitigate problems between multiple systems working on this project.
+
+## Install virtualenv
+
+```bash
+$ cd ~
+$ mkdir .virtualenvs
+$ sudo pip install virtualenv
+```
+Verify virtualenv is installed
+
+```bash
+$ which virtualenv
+```
+
+## Create virtual environment
+
+Navigate inside the `python` directory and run the following
+
+```bash
+$ virtualenv -p python3 ~/.virtualenvs/aida-interchange-format
+$ source ./virtualenvs/aida-interchange-format/bin/activate
```

Comment:
This needs to be `~/.virtualenvs`, or `./env` if we go that route.

Reply:
Fixed.